src/q3/methodnum-FSAB1104/summary/methodnum-FSAB1104-summary.tex

```diff
@@ -782,43 +794,45 @@ \subsection{Méthodes d'Adams-Bashforth-Moulton}
 d'Adams-Bashforth et lorsqu'elle est explicite, on l'appelle
 la méthode d'Adams-Moulton.
 
-La méthode est une \emph{méthode à pas lié}.
-Pour utiliser des coefficients calculés à priori,
+La méthode est une \emph{méthode à pas liés}.
+Pour utiliser des coefficients calculés a priori,
 il faut faire attention de ne pas modifier le pas $h$.
 
 L'ordre des méthodes est $n+1$ pour les méthodes d'Adams-Bashforth et
 $n+2$ pour les méthodes d'Adams-Moulton.
 
 \subsection{Méthodes de Gear}
 Pour les méthodes de Gear,
-on calculer le polynôme d'interpolation passant par les points de
+on calcule le polynôme d'interpolation passant par les points de
 $X_{i-n+1}$ à $X_{i+1}$ et on impose que sa dérivée valle $F_i$ ou
 $F_{i+1}$.
 Si on impose qu'elle valle $F_i$, c'est une méthode explicite,
 sinon, c'est une méthode implicite.
-Pour la méthode de Gear, utiliser $F_{i+1}$ donc
-\[ \sum_{j=0}^n U_{i+1-j}\phi'_j(X_{i+1}) \approx F_{i+1}. \]
+Pour la méthode de Gear, on utilise $F_{i+1}$ donc
+\[ \sum_{j=0}^n U_{i+1-j}\phi'_j(X_{i+1}) \approx F_{i+1}, \]
 ce qui nous donne
 \[ U_{i+1} = \frac{1}{\phi'(X_{i+1})}
 \left(-\sum_{j=1}^n U_{i+1-j}\phi'_j(X_{i+1}) + F_{i+1}\right). \]
-Le degré de la méthode est $n$, c'est à dire le nombre de $U_j$ utilisés
+Le degré de la méthode est $n$, c'est-à-dire le nombre de $U_j$ utilisés
 sans compter $U_{i+1}$.
 
-\subsection{Méthode de Leapfrog}
-La méthode de Leapfrog combine astucieusement deux développement de Taylor
-\begin{align*}
-  U_{i+1} & = U_i + hU_i' + \frac{h^2}{2}U_i'' + \bigoh(h^3)\\
-  U_{i-1} & = U_i - hU_i' + \frac{h^2}{2}U_i'' + \bigoh(h^3)\\
-  U_{i+1} - U_{i-1} & = 2hU_i' + \bigoh(h^3)\\
-  U_{i+1} & = U_{i-1} + 2hF_i + \bigoh(h^3)
-\end{align*}
+\subsection{Méthode Leapfrog}
+La méthode Leapfrog combine astucieusement deux développements de Taylor
+\begin{equation*}
+\begin{array}{crcl}
+  &U_{i+1} & = & U_i + hU_i' + \frac{h^2}{2}U_i'' + \bigoh(h^3)\\
+  &U_{i-1} & = & U_i - hU_i' + \frac{h^2}{2}U_i'' + \bigoh(h^3)\\
+  \iff& U_{i+1} - U_{i-1} & = & 2hU_i' + \bigoh(h^3)\\
+  \iff& U_{i+1} & = & U_{i-1} + 2hF_i + \bigoh(h^3).
+\end{array}
```

Comment:
Just in case, il me semble que tu peux aligner suivant plusieurs colonnes avec `align`.

Reply:
Pas avec `align`, mais avec `alignat`, oui. Le seul problème, c'est que `alignat` ne tient pas compte des espacements d'opérateurs (ou en tout cas, c'est pas immédiat).